iocore/net/P_UDPPacket.h

```diff
@@ -246,6 +246,41 @@ new_incoming_UDPPacket(struct sockaddr *from, char *buf, int len)
 }
 
 TS_INLINE UDPPacket *
+new_incoming_UDPPacket(struct sockaddr *from, IOBufferBlock *block, int len)
+{
+  (void)len;
+  UDPPacketInternal *p = udpPacketAllocator.alloc();
+  IOBufferBlock *body;
+
+  p->in_the_priority_queue = 0;
+  p->in_heap               = 0;
+  p->delivery_time         = 0;
+  ats_ip_copy(&p->from, from);
+
+  while (block) {
+    body = block->clone();
+    p->append_block(body);
+    block = block->next.get();
+  }
+
+  return p;
+}
+
+TS_INLINE UDPPacket *
+new_incoming_UDPPacket(struct sockaddr *from, Ptr<IOBufferBlock> &buf)
```

Comment:
In the prototype of this function, this `buf` is called `block`. It'd be great if you pick out one of them for consistency :)

Reply:
pick `block` as the name.